migrations/sql/V2021.04.06.14.53__mine_type_migration_to_permit_level.sql

```diff
@@ -0,0 +1,334 @@
+DROP INDEX IF EXISTS mine_guid_mine_tenure_type_code_permit_guid_active_uniqeness;
+
+CREATE OR REPLACE FUNCTION max (uuid, uuid)
+RETURNS uuid AS $$
+BEGIN
+	IF $1 IS NULL OR $1 < $2 THEN
+		RETURN $2;
+	END IF;
+
+	RETURN $1;
+END;
+$$ LANGUAGE plpgsql;
+
+
+CREATE AGGREGATE max (uuid)
+(
+    sfunc = max,
+    stype = uuid
+);
+
+DROP TABLE IF EXISTS multiple_permits;
+CREATE TEMP TABLE multiple_permits AS 
+	SELECT mine_guid, permit_guid FROM mine_permit_xref mpx JOIN permit p ON mpx.permit_id = p.permit_id WHERE mine_guid IN (
+	SELECT mine_guid FROM mine_permit_xref mpx JOIN permit p ON mpx.permit_id = p.permit_id
+	WHERE p.permit_status_code = 'O'
```

Comment:
Why do we only want to do this for open permits? Why not draft or closed?

Reply:
Sure, let's do it for all types.